grammar.js

```diff
@@ -775,7 +806,7 @@ module.exports = grammar({
 
     _property_name: $ => choice($.identifier, $.reserved_identifier, $.string, $.number),
 
-    reserved_identifier: $ => choice('get', 'set', 'async')
+    reserved_identifier: $ => choice('get', 'set', 'async', 'abstract', 'module', 'interface'),
```

Comment:
Should these just be added in the typescript grammar? I'd think it'd be ok to just let these words be parsed as regular `identifiers` in regular JavaScript.